rskj-core/src/main/java/org/ethereum/core/TransactionExecutor.java

```diff
@@ -195,6 +207,14 @@ private boolean init() {
         return true;
     }
 
+    private boolean isInitCodeSizeInvalidForTx(Transaction tx) {
+        int initCodeSize = getLength(tx.getData());
+
+        return tx.isContractCreation()
+                && activations.isActive(ConsensusRule.RSKIP438)
+                && initCodeSize > Constants.getMaxInitCodeSize();
+    }
```

Comment:
good one 👍 I believe we should also make this validation in mempool, in `TxPendingValidator#isValid()`, perhaps somewhere around [this](https://github.com/rsksmart/rskj/blob/925fdf6c85de61a622fb69e70f0f09fa418334ad/rskj-core/src/main/java/co/rsk/net/handler/TxPendingValidator.java#L84)

Reply:
Ok, I will do it.